### PR TITLE
MaaS Queens fixes

### PR DESCRIPTION
**Pull request by tonytan4ever**

### CODE REVIEW

File: tests/aio-create.sh [hunk 1]
```diff
@@ -201,6 +201,10 @@ pushd /opt/openstack-ansible
     # Pin flask so it stops breaking xenial and other versions
     echo "Flask==0.12.2" >> /opt/openstack-ansible/global-requirement-pins.txt
     enable_ironic
+  
+  elif [ "${RE_JOB_SCENARIO}" == "queens" ]; then
+    git checkout "stable/queens"  # Branch checkout of Pike (Current Stable)
```
Comment (hmakkapati) [inline tests/aio-create.sh]: The comment says `Pike`, probably a copy-pasta error?

Comment (tonytan4ever) [reply tests/aio-create.sh]: Done

**Comment by CrashenX** [comment]
@tonytan4ever  @hmakkapati Why are all the deploy checks still pending?

**Comment by tonytan4ever** [comment]
@CrashenX:  Addressed @hmakkapati's comment.  All gates are re-running.

**Comment by hmakkapati** [comment]
recheck_xenial_queens_deploy

**Comment by hmakkapati** [comment]
recheck_xenial_newton_deploy

**Comment by hmakkapati** [comment]
recheck_xenial_newton_deploy

**Comment by tonytan4ever** [comment]
recheck_xenial_queens_deploy

**Comment by hmakkapati** [comment]
rechec_xenial_newton_deploy

**Comment by hmakkapati** [comment]
recheck_xenial_newton_deploy

**Pull request merged**